packages/strapi/lib/services/__tests__/entity-validator.test.js

```diff
@@ -33,12 +33,6 @@ describe('Entity validator', () => {
       });
 
       it('Returns data on valid input', async () => {
-        global.strapi = {
```

Comment:
When I was talking about refactoring badRequest declaration, I was more thinking about something like `beforeEach(() => { global.strapi = { errors: { badRequest: jest.fn(); } } })` and `afterEach(() => { global.strapi.errors.badRequest.mockClear()})`. What do you think?